pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -133,9 +133,8 @@ public HelixBrokerStarter(PinotConfiguration brokerConf, String clusterName, Str
 
   private int inferPort() {
     return Optional.ofNullable(_brokerConf.getProperty(Helix.KEY_OF_BROKER_QUERY_PORT)).map(Integer::parseInt)
-        .orElseGet(() -> _listenerConfigs.stream().findFirst().map(ListenerConfig::getPort).orElseThrow(() ->
-            new IllegalStateException(String.format("Requires at least one ingress config or '%s'",
-                Helix.KEY_OF_BROKER_QUERY_PORT))));
+        .orElseGet(() -> _listenerConfigs.stream().findFirst().map(ListenerConfig::getPort)
```

Comment:
With the current way of extracting the listener config, we should be able to always use the first one within the listener config?

Reply:
yes, changed